apps/web-giddh/src/app/inventory/components/aside-transfer-pane/aside-transfer-pane.component.scss

```diff
@@ -18,14 +18,18 @@
 }
 
 :host.in #close {
-    display: block;
-    position: fixed;
+    position: absolute;
     left: auto;
-    right: 472px;
-    top: 0;
-    z-index: 5;
+    right: 10px;
+    z-index: 9999999;
```

Comment:
Please check the z-index of close button in mobile view in  comparison to modal, date range